### PR TITLE
chore: added get record versions endpoint

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/DAL/repositories/recordRepository.ts [hunk 1]
```diff
@@ -53,4 +53,13 @@ export class RecordRepository extends Repository<RecordEntity> {
     });
     return res.map((entity) => this.recordConvertor.entityToModel(entity));
   }
+
+  public async getRecordVersions(req: IFindRecordRequest): Promise<any> {
```
Comment (asafMasa) [inline src/DAL/repositories/recordRepository.ts:57]: lint

File: openapi3.yaml [hunk 2]
```diff
@@ -124,6 +124,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error'
+  /records/find/versions:
```
Comment (asafMasa) [inline openapi3.yaml:127]: add test
1. for valid request
2. for invalid request

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: openapi3.yaml [hunk 3]
```diff
@@ -124,6 +124,33 @@ paths:
...
+              $ref: '#/components/schemas/findRecordModel'
+      responses:
+        '200':
+          description: retrieved matching records
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/findRecordModel'
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply openapi3.yaml:147]: fixed

File: src/DAL/repositories/recordRepository.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/DAL/repositories/recordRepository.ts:57]: fixed

File: openapi3.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply openapi3.yaml:127]: valid request exists => 'should return 200 and array of exists record versions catalog'

File: tests/integration/records/records.spec.ts [hunk 4]
```diff
@@ -229,21 +229,38 @@ describe('records', () => {
...
+      } as unknown as RecordEntity;
+      findMock.mockResolvedValue([testEntity]);
+      const req = {
+        metadata: {
+          productId: 'test-id',
+          productType: 'Orthophoto',
+        },
+      };
+      const response = await requestSender.getRecordVersions(req);
+      const expectedResponse = ['12.36'];
```
Comment (asafMasa) [inline tests/integration/records/records.spec.ts:255]: add row "expect(response).toSatisfyApiSpec();"

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

**Pull request merged**